rest/class.wp-super-cache-rest-get-settings.php

```diff
@@ -172,8 +172,7 @@ protected function get_is_preload_active() {
 	 */
 	protected function get_minimum_preload_interval() {
 		global $wpdb;
-		$posts_count = wp_count_posts();
-		$count = $posts_count->publish;
+		$count = wpsc_post_count();
```

Comment:
Maybe, we can use `$count = $this->get_post_count();` instead `$count = wpsc_post_count();` It seems as more flexible solution.